x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -0,0 +1,216 @@
+outputs:
+  default:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    # Not supported at first
+    queue:
+      type: disk
+
+  long_term_storage:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
```

Comment:
are we assuming everything is elasticsearch or do we support type still

Reply:
Lets keep the type support that we have, we are so use to target elasticsearch I think this felt into a crack. so `type: "elasticsearch`

Reply:
I just added the type to this PR.